packages/web-twig/src/Resources/components/Link/Link.twig

```diff
@@ -26,8 +26,8 @@
     {{ mainProps(props) }}
     href="{{ _href }}"
     {{ classProp(_classNames) }}
-    {{ _onClickAttr | raw }}
-    {{ _targetAttr | raw }}
+    {{ _onClickAttr  }}
+    {{ _targetAttr  }}
     {{ _titleAttr | raw }}
```

Comment:
@kdosiodjinud Here is a behavior that I do not understand:

Input to the `title` is `test title`. Without using `raw` the output is only `test`. With using `| raw | escape` the output is still only the first word. It seems to me like the escaping functionality will remove everything behind blank space. 🤔